### PR TITLE
delete product

### PR DESCRIPTION
**Pull request by irenezhengg**

### CODE REVIEW

**Review by Ming119 — APPROVED** [review]

File: app/templates/base.html [hunk 1]
```diff
@@ -16,37 +16,37 @@
...
+                    <li class="nav-item"><a id="UserProfile" href='{{ url_for('user.profile', user_id=current_user.user_id) }}' class="nav-link"><i class="fas fa-user me-1"></i>{{ current_user.username }}</a></li>
+                    <li class="nav-item"><a id="Logout" href='{{ url_for('index.logout') }}' class="nav-link"><i class="fas fa-sign-out-alt me-1"></i>Logout</a></li>
                 </ul>
             </div>
             {% else %}
             <div class="">
                 <ul class="navbar-nav">
-                    <li class="nav-item"><a href='{{ url_for('index.login') }}' class="nav-link"><i class="fas fa-sign-in-alt me-1"></i>Login</a></li>
-                    <li class="nav-item"><a href='{{ url_for('index.register') }}' class="nav-link"><i class="fas fa-user-plus me-1"></i>Sign-up</a></li>
+                    <li class="nav-item"><a id="login" href='{{ url_for('index.login') }}' class="nav-link"><i class="fas fa-sign-in-alt me-1"></i>Login</a></li>
```
Comment (Ming119) [inline app/templates/base.html:48]: Please change to `id="Login"`

**Review by FuChiKei — APPROVED** [review]

File: app/controllers/productController.py [hunk 2]
```diff
@@ -105,7 +105,8 @@ def details(product_id):
                     productDescription = product.description,
                     price              = product.price,
                     quantity           = product.quantity,
-                    category           = product.category_id
+                    category           = product.category_id,
+                    categories = categories
                 )
         form.category.choices = [(category.category_id, category.name) for category in categories]
```
[the remark this answers is not in the document]
Comment (irenezhengg) [reply app/controllers/productController.py:111]: done

File: app/templates/base.html [hunk 3]
```diff
@@ -16,37 +16,37 @@
 <body>
     <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
         <div class="container d-flex justify-content-between">
-            <a class="navbar-brand" href="{{ url_for('index.index') }}">
+            <a id="home" class="navbar-brand" href="{{ url_for('index.index') }}">
```
[the remark this answers is not in the document]
Comment (irenezhengg) [reply app/templates/base.html:19]: done

File: app/templates/base.html (hunk 1 above)
Comment (irenezhengg) [reply app/templates/base.html:48]: done

**Review by Ming119 — APPROVED** [review]

**Review by NTUT-LUNA — APPROVED** [review]
LGTM

**Pull request merged**